doc-content/enterprise-only/integration/sso-remote-services-securing-proc.adoc

```diff
@@ -19,6 +19,15 @@ image::integration/kie_git_client_settings.png[]
 .. Click the *Installation* tab at the top of the client configuration screen and choose *Keycloak OIDC JSON* as a *Format Option*.
 .. Click *Download*.
 . Move the downloaded JSON file to an accessible directory in the server's file system or add it to the application class path.
+. In the `_EAP_HOME_/standalone/configuration/standalone-full.xml` file, under the `<system-properties>` tag, add the following system property:
++
+[source,xml]
+----
+<property name="rg.uberfire.ext.security.keycloak.keycloak-config-file" value="$EAP_HOME/kie-git.json"/>
```

Comment:
actual property name is '"org.uberfire.ext.security.keycloak.keycloak-config-file", we are missing an extra 'O' @mramendi